-------------------------------------------------------------------------------
Test set: Entity.ValidatorTest.EmployeeValidationTest
-------------------------------------------------------------------------------
Tests run: 4, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 5.164 sec <<< FAILURE! - in Entity.ValidatorTest.EmployeeValidationTest
persistirEmployeeInvalido(Entity.ValidatorTest.EmployeeValidationTest)  Time elapsed: 0.007 sec  <<< ERROR!
javax.persistence.PersistenceException: 
Exception [EclipseLink-7060] (Eclipse Persistence Services - 2.5.2.v20140319-9ad6abd): org.eclipse.persistence.exceptions.ValidationException
Exception Description: Cannot acquire data source [jdbc/ductum].
Internal Exception: javax.naming.NamingException: Lookup failed for 'jdbc/ductum' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.url.pkgs=com.sun.enterprise.naming, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl} [Root exception is javax.naming.NameNotFoundException: ductum not found]
Caused by: org.eclipse.persistence.exceptions.ValidationException: 

Exception Description: Cannot acquire data source [jdbc/ductum].
Internal Exception: javax.naming.NamingException: Lookup failed for 'jdbc/ductum' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.url.pkgs=com.sun.enterprise.naming, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl} [Root exception is javax.naming.NameNotFoundException: ductum not found]
Caused by: javax.naming.NamingException: Lookup failed for 'jdbc/ductum' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.url.pkgs=com.sun.enterprise.naming, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl}
Caused by: javax.naming.NameNotFoundException: ductum not found

persistirEmployeeInvalido(Entity.ValidatorTest.EmployeeValidationTest)  Time elapsed: 0.007 sec  <<< ERROR!
java.lang.NullPointerException

atualizarEmployeeInvalido(Entity.ValidatorTest.EmployeeValidationTest)  Time elapsed: 0.001 sec  <<< ERROR!
javax.persistence.PersistenceException: 
Exception [EclipseLink-7060] (Eclipse Persistence Services - 2.5.2.v20140319-9ad6abd): org.eclipse.persistence.exceptions.ValidationException
Exception Description: Cannot acquire data source [jdbc/ductum].
Internal Exception: javax.naming.NamingException: Lookup failed for 'jdbc/ductum' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.url.pkgs=com.sun.enterprise.naming, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl} [Root exception is javax.naming.NameNotFoundException: ductum not found]
Caused by: org.eclipse.persistence.exceptions.ValidationException: 

Exception Description: Cannot acquire data source [jdbc/ductum].
Internal Exception: javax.naming.NamingException: Lookup failed for 'jdbc/ductum' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.url.pkgs=com.sun.enterprise.naming, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl} [Root exception is javax.naming.NameNotFoundException: ductum not found]
Caused by: javax.naming.NamingException: Lookup failed for 'jdbc/ductum' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.url.pkgs=com.sun.enterprise.naming, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl}
Caused by: javax.naming.NameNotFoundException: ductum not found

atualizarEmployeeInvalido(Entity.ValidatorTest.EmployeeValidationTest)  Time elapsed: 0.001 sec  <<< ERROR!
java.lang.NullPointerException

